changelog

```diff
@@ -10,5 +10,7 @@ cgrates (0.10) UNRELEASED; urgency=medium
   [ Voivozeanu Teofil ]
   * [AgentRequest] Add support for *group type and correctly overwrite
     the values in case of *variable
+  * [EventReader] Correctly populate ConcurrentRequest from config in
+    EventReader
```

Comment:
no need of the second EventReader in the comment since you already have it in header